src/main/java/org/spongepowered/api/event/cause/entity/damage/source/DamageSource.java

```diff
@@ -41,6 +41,7 @@
  * the {@link Cause} of the event. Any additional modifiers that "aid" the
  * {@link Cause} of the event will be listed subsequently.</p>
  */
+@Deprecated // TODO Temporary as a marker to myself that I'd like to remove this
```

Comment:
I was just wanting to remove all wrapper causes, DamageSource is a little different though as it is for plugins to pass in this type information to the `Entity#damage` method. The subclasses should probably go though.